concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/updates/queues/TimeParameters.java

```diff
@@ -19,10 +19,10 @@ public final class TimeParameters {
     /**
      * Mint rate per payday (as a proportion of the extant supply).
      */
-    private final MintRate mintPerPayday;
+    private final double mintPerPayday;
```

Comment:
Well this is not entirely accurate. Why the change?

At least we need to document a bit that there might be rounding issues.

Reply:
I have documented the caveat here. 
The reason for the change was a bugfix, as the 'mintPerPayday' is being returned as a number as opposed to a struct with an exponent and mantissa.